packages/components/src/ui/color-picker/styles.ts

```diff
@@ -6,13 +6,37 @@ import styled from '@emotion/styled';
 /**
  * Internal dependencies
  */
+import NumberControl from '../../number-control';
 import InnerSelectControl from '../../select-control';
 import InnerRangeControl from '../../range-control';
 import { StyledField } from '../../base-control/styles/base-control-styles';
+import { ThumbWrapper } from '../../range-control/styles/range-control-styles'
 import { space } from '../utils/space';
+import {
+	BackdropUI,
+	Container as InputControlContainer,
+} from '../../input-control/styles/input-control-styles';
+import InputControl from '../../input-control';
+
+export const InputWrapper = styled( InputControl )`
+	${ InputControlContainer } {
+		padding: 5px 8px;
```

Comment:
I'm still wrapping my head around .ts files, but when I see bespoke paddings and margins like these it gets me a little worried about code drift. What's the best solution for keeping things the same here? I understand re-used SCSS variables (like the `$grid-unit`s we have) isn't possible here. Is it more of a matter of updating the source component used?

Reply:
After switching to Emotion, we've been using the `space` util (`packages/components/src/ui/utils/space.ts`) in combination with some shared configuration (`packages/components/src/utils/config-values.js`) — we should probably leverage them here as well, instead of these hardcoded values

(also cc'ing @sarayourfriend in case I missed anything)